### DISCUSSION TITLE
Passing parameters to labgrid-exporter container

### DISCUSSION THREAD

**Discussion by userid0x0** [labgrid-project/labgrid#1034] [General]
Hi,
I think the labgrid-exporter docker container mixes up `CMD` and `ENTRYPOINT`. The script https://github.com/labgrid-project/labgrid/blob/master/dockerfiles/exporter/entrypoint.sh foresees parameter passing from the commandline (note the `$@`).
When I do
```
  $ podman run ... labgrid-exporter -n my-docker-host
```
I get an error that the command `-n....` wasn't found. But this works
```
  $ podman run ... labgrid-exporter /entrypoint.sh -n my-docker-host
```
. I think the `CMD` statement in the dockerfile should be replaced with `ENTRYPOINT` (https://github.com/labgrid-project/labgrid/blob/master/dockerfiles/Dockerfile#L126)

**Comment by Bastian-Krause** [comment]
Maybe @krevsbech can comment on this?
